src/styles/container.scss

```diff
@@ -0,0 +1,9 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 50px auto 0;
+  padding: 0 120px;
+  box-sizing: content-box;
+  max-width: 1024px;
+}
```

Comment:
maybe this is normal, but... I like grid.
grid - has a strict structure that you specify and you sure that your code don't break.

you could use this code

.container {
    display: grid;
    grid-template-columns: repeat(4,1fr);
    margin: 50px auto 0;
    justify-content: center;
    padding: 0 120px;
    max-width: 1024px;
}

and clean 
.card--location {
  margin-bottom: 50px;
  margin-left: 48px;
}

.card--location:nth-child(4n+1) {
  margin-left: 0;
}

oh my , you need full refactor your logic, i write in commit, if shortly
< ul class="container">
< li class="container-item">
 < div class="card card--location">
  ...
 </ div>
< li>
</ ul>


I agree with you, maybe you don't understand. to message me in slack and I will help you